Sources/TokamakCore/MountedViews/MountedCompositeView.swift

```diff
@@ -44,6 +44,10 @@ final class MountedCompositeView<R: Renderer>: MountedView<R>, Hashable {
   override func mount(with reconciler: StackReconciler<R>) {
     let childBody = reconciler.render(compositeView: self)
 
+    if let appearanceAction = view.view as? AppearanceActionProtocol {
```

Comment:
Will this work if there's another modifier wrapped around the appearance action modifier?

Reply:
I think it does work, at least in my testing where both `onAppear` and `onDisappear` are chained. The wrapping modifier will get this function called separately. In my understanding wrapped modifiers are peeled like an onion by this function recursively.